src/main/java/net/sf/mzmine/datamodel/PeakStatus.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
What is this file for? I don't see it it in the main MZmine project.

https://github.com/mzmine/mzmine2/tree/master/src/main/java/net/sf/mzmine/datamodel